### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by odedlaz**
updated the README to include a section explaining how to use an external configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ A pre-built image is available on [Docker Hub](https://registry.hub.docker.com/u
...
+vault binds to 127.0.0.1 by default, make sure you bind vault's listener to to 0.0.0.0.
+##### development server
+when running the development server, vault binds to localhost by default. A possible workaround would be to create the following vault config:
+```
+listener "tcp" {
+  address = "0.0.0.0:9000"
+  tls_disable = 1
+}
+```
+and than pass the configuration to vault. for example, if you saved the mentioned configuration at /tmp/vault.conf:
```
Comment (greg0ire) [inline README.md]: "and than" => "and then"

**Comment by greg0ire** [comment]
The commit message shoud read "document how to use an external configuration". "Update file.ext" is not very informative and is too low level.

File: README.md (hunk 1 above)
Comment (odedlaz) [inline README.md]: fixed.

**Comment by odedlaz** [comment]
@greg0ire I updated the commit message and fixed the typo. anything else?

**Comment by greg0ire** [comment]
Good for me

**Comment by odedlaz** [comment]
so merge it @greg0ire  :smile:

**Comment by greg0ire** [comment]
I would if I could, man, I would if I could…

**Comment by greg0ire** [comment]
ping @cgswong

**Pull request merged**